docker-compose.buildkite.yml

```diff
@@ -8,6 +8,11 @@ services:
 
   app-production: &production
     build: .
+    environment:
+      - BUILD_HASH=${BUILDKITE_COMMIT}
```

Comment:
The question in Dockerfile **or** should these be made build args (and in app-staging)?